intg/src/main/java/org/apache/atlas/model/impexp/AtlasAsyncImportRequest.java

```diff
@@ -0,0 +1,383 @@
+    }
+
+    @JsonIgnore
+    public Map<String, Object> getImportMinInfo() {
+        Map<String, Object> minInfoResponse = new HashMap<>();
+
+        String importId = this.getImportId();
+        long timestamp = this.receivedAt;
+        String isoDate = toIsoDate(new Date(timestamp));
+
+        minInfoResponse.put("importId", importId);
+        minInfoResponse.put("status", status);
+        minInfoResponse.put("importRequestReceivedAt", isoDate);
+        minInfoResponse.put("importRequestReceivedBy", atlasImportResult.getUserName());
+
+        return minInfoResponse;
+    }
+
+    private String toIsoDate(Date value) {
+        final TimeZone tz = TimeZone.getTimeZone("UTC");
+        final DateFormat df = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'");
+        df.setTimeZone(tz);
+        return df.format(value);
+    }
+
+    @Override
+    public String toString() {
+        return toString(new StringBuilder()).toString();
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (!(o instanceof AtlasAsyncImportRequest)) {
+            return false;
+        }
+        if (!super.equals(o)) {
+            return false;
+        }
+        AtlasAsyncImportRequest that = (AtlasAsyncImportRequest) o;
+        return Objects.equals(atlasImportResult, that.atlasImportResult) &&
+                Objects.equals(importId, that.importId) &&
+                Objects.equals(status, that.status) &&
+                Objects.equals(importDetails, that.importDetails) &&
+                Objects.equals(requestId, that.requestId) &&
+                Objects.equals(receivedAt, that.receivedAt) &&
+                Objects.equals(stagedAt, that.stagedAt) &&
+                Objects.equals(startedProcessingAt, that.startedProcessingAt) &&
+                Objects.equals(completedAt, that.completedAt);
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(super.hashCode(), requestId, atlasImportResult,
+                importId, status, importDetails, receivedAt, stagedAt, startedProcessingAt, completedAt);
+    }
+
+    @Override
+    protected StringBuilder toString(StringBuilder sb) {
+        sb.append(", atlasImportResult=");
+        if (atlasImportResult == null) {
+            sb.append("null");
+        } else {
+            sb.append(atlasImportResult);
+        }
+        sb.append(", requestId=").append(requestId);
+        sb.append(", importId=").append(importId);
+        sb.append(", status=").append(status);
+        sb.append(", receivedAt=").append(receivedAt);
+        sb.append(", stagedAt=").append(stagedAt);
+        sb.append(", startedProcessingAt=").append(startedProcessingAt);
+        sb.append(", completedAt=").append(completedAt);
+        sb.append(", importDetails=").append(importDetails);
+
+        return sb;
+    }
+
+    @JsonIgnoreProperties(ignoreUnknown = true)
+    public static class ImportDetails {
```

Comment:
- add `implements Serializable`
- `failures` is marked as `final`. How does this work drring deserialization?
- `failures` is initialized with `ConcurrentHashMap`. Is this necessary? Json deserializer might assign a Hashmap instance.